lib/workflowmgr/slurmbatchsystem.rb

```diff
@@ -385,8 +386,9 @@ def refresh_jobqueue(jobids)
           queued_jobs,errors,exit_status=WorkflowMgr.run4("squeue --jobs=#{joblist} -M all -t all -O jobid:40,username:40,numcpus:10,partition:20,submittime:30,starttime:30,endtime:30,priority:30,exit_code:10,state:30,name:200",45)
         end
 
-        # Raise SchedulerDown if the command failed
-        raise WorkflowMgr::SchedulerDown,errors unless exit_status==0
+        # Don't raise SchedulerDown if the command failed, otherwise
+        # jobs that have moved to sacct will be missed
+        # raise WorkflowMgr::SchedulerDown,errors unless exit_status==0
```

Comment:
Please update the rescue clause on line 396 to remove  the catching of `WorkflowMgr::SchedulerDown` since it is no longer being raised.

```rescue Timeout::Error,WorkflowMgr::SchedulerDown```